tests/acceptance/tests/basic_dsc_resources/waitforall/negative/waitforall_blacklist.rb

```diff
@@ -18,7 +18,7 @@
 dsc_manifest = ERB.new(File.read(dsc_manifest_template_path), 0, '>').result(binding)
 
 # Verify
-error_msg = /Error:.*Invalid resource type dsc_waitforall/
+error_msg = /Error:.*resource.*dsc_waitforall/
```

Comment:
We could, I thought about it. I decided against it in case someone comes along and adds `'Invalid resource'` or something to that effect in a similar manner to what was done with `type` to `'type'`